app/MMB/src/scenes/help/Help.js

```diff
@@ -0,0 +1,94 @@
+// @flow
+
+import * as React from 'react';
+import idx from 'idx';
+import { ScrollView, Platform, Linking } from 'react-native';
+import { TextIcon, StyleSheet, Color } from '@kiwicom/mobile-shared';
+import { Translation, Alert } from '@kiwicom/mobile-localization';
+import {
+  MenuItem,
+  MenuGroup,
+  SeparatorFullWidth,
+  SeparatorTrimmed,
+  type RouteNamesType,
+  type NavigationType,
+} from '@kiwicom/mobile-navigation';
+
+import type { HelpContainerQueryResponse } from './__generated__/HelpContainerQuery.graphql';
```

Comment:
It would be better to create your custom Relay fragment in this component instead.

Reply:
Sure, I will do it.